server/graphql/v1/types.js

```diff
@@ -1262,6 +1262,8 @@ export const TierStatsType = new GraphQLObjectType({
             return req.loaders.Tier.totalMonthlyDonations.load(tier.id);
           } else if (tier.interval === intervals.YEAR) {
             return req.loaders.Tier.totalYearlyDonations.load(tier.id);
+          } else if (tier.interval === intervals.FLEXIBLE) {
+            return req.loaders.Tier.totalDonated.load(tier.id);
```

Comment:
According to the description, this does not seem like a correct definition:
> How much money is given for this tier for each tier.interval (monthly/yearly)

This will return the total amount donated for this tier, which is fundamentally different from the sum of the total active recurring contributions. To solve that, an option is to introduce a `totalRecurringDonations` loader that will do:
> totalMonthlyContributions + (totalYearlyContributions / 12)

We can then update the field's description to:
```diff
- How much money is given for this tier for each tier.interval (monthly/yearly)
+ How much money is given for this tier for each tier.interval (monthly/yearly). For flexible tiers, this amount is a monthly average of contributions amount, taking into account both yearly and monthly subscriptions.
```

Reply:
@Betree  Something like this for flexible - 

```
        SELECT SUM(total) from (
            SELECT o."TierId" AS "TierId", COALESCE(SUM(s."amount")/12, 0) AS "total"
            FROM "Orders" o
            INNER JOIN "Subscriptions" s ON o."SubscriptionId" = s.id
            WHERE "TierId" IN (?)
            AND s."isActive" = TRUE
            AND s."interval" = 'year'
            GROUP BY "TierId"
            UNION ALL
            SELECT o."TierId" AS "TierId", COALESCE(SUM(s."amount"), 0) AS "total"
            FROM "Orders" o
            INNER JOIN "Subscriptions" s ON o."SubscriptionId" = s.id
            WHERE "TierId" IN (?)
            AND s."isActive" = TRUE
            AND s."interval" = 'month'
            GROUP BY "TierId"
        );
```

Reply:
Added a new `totalRecurringDonations` loader 👍🏻